lib/avoid_forbidden_namespaces.ex

```diff
@@ -7,17 +7,20 @@ defmodule Nicene.AvoidForbiddenNamespaces do
 
   @explanation [check: @moduledoc]
 
-  use Credo.Check, base_priority: :high, category: :refactoring
+  use Credo.Check,
+    base_priority: :high,
+    category: :refactoring,
+    param_defaults: [from: nil, forbid: nil]
```

Comment:
Custom parameters should be listed in `param_defaults` to avoid credo giving an "unknown parameter ..." warning